babel_godot.py

```diff
@@ -81,6 +82,16 @@ def check_translate_property(property):
                 value = match.group(2)
                 keyword = check_translate_property(property)
                 if keyword:
-                    value = _godot_unquote(value)
+                    value = _godot_unquote(value.strip())
```

Comment:
I'm not sure why the issue with the tscn happens, nor if it is an isolated case. I don't think that fix can cause issues, but let me know if there is any.